core/driver/sources/src/main/scala/org/virtuslab/ideprobe/IdeProbeFixture.scala

```diff
@@ -4,18 +4,22 @@ import java.util.concurrent.{SynchronousQueue, ThreadPoolExecutor, TimeUnit}
 import scala.concurrent.ExecutionContext
 import scala.util.Try
 
-trait IdeProbeFixture {
-  protected implicit val ec: ExecutionContext = {
+object IdeProbeFixture {
+  def defaultEc(): ExecutionContext = {
     val service = new ThreadPoolExecutor(0, Integer.MAX_VALUE, 10L, TimeUnit.SECONDS, new SynchronousQueue[Runnable])
     service.allowCoreThreadTimeOut(true)
```

Comment:
Would be nice to document why this is needed while you're here before this knowledge is lost **f o r e v e r**

Reply:
Done